bundles/org.openhab.binding.lifx/src/main/java/org/openhab/binding/lifx/internal/util/LifxMessageUtil.java

```diff
@@ -117,6 +128,13 @@ public static int percentTypeToKelvin(PercentType temperature, TemperatureRange
                 temperatureRange.getMaximum() - (temperature.floatValue() * (temperatureRange.getRange() / 100)));
     }
 
+    public static int quantityTypeToKelvin(QuantityType temperature, TemperatureRange temperatureRange) {
+        QuantityType<?> asKelvin = temperature.toInvertibleUnit(Units.KELVIN);
+        assert asKelvin != null;
+
+        return asKelvin.toBigDecimal().intValue();
```

Comment:
@cweitkamp used `convertedCommand.intValue()` in the hue PR/
Is there a difference between `xxx.intValue()` and `xxx.toBigDecimal().intValue()` ?

Reply:
Please also don't use assertions.

Reply:
🤦  I totally looked for QuantityType().intValue(), and didn't see it.